.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
```

Comment:
Aqui seria melhor ser qualquer branch não? Por que senão só vai ser executado no merge, e o ideal é avaliar ainda nos PRs para não ser necessário nenhum ajuste no momento do merge.

Reply:
acredito que só aberto PR para develop e master, mas se for o caso eu coloco para todos branches.

Reply:
Putz, tem razão. Desconsidere!